lib/decidim/gem_manager.rb

```diff
@@ -207,6 +209,13 @@ def root
       def version_file
         File.join(root, ".decidim-version")
       end
+
+      def replace_antora_version
+        antora_conf = File.join(root, "docs/antora.yml")
+        docs_version = version
```

Comment:
As you added the ternary operation, I think this assignment is unnecessary.

In the line below, you'll also need to change the `docs_version.match?` to `version.match?`.

Reply:
Of course! Changed at 4ddb64d152b7395d3d02f6acee07d883ef9ed5aa